### DISCUSSION TITLE
Change the name of this project

### DISCUSSION THREAD

**Discussion by allisoncfigus** [figus-cli/figus#29] [Ideas]
I am Allison Figus and I use Paintbrush to create wireframes.
Figus is a Latin word meaning "Loyalty to family" and it is also an old Italian last name pre-dating Christ.
This is polite and friendly "hahah" opportunity for you to just change your name.
I can and will get Ted Figus to do the real legal filing if you don't just stop.

**Comment by omridevk** [comment]
Hey,

Thank you for your contribution and suggestion. To better understand, you like the project and are proposing I change my last name to Figus? I like "Loyalty" and actually have some Italian Heritage so - cool original idea.

I'll see how much work it is and let you know, thanks again ❤️

**Reply by allisoncfigus** [reply]
https://www.dmlp.org/legal-guide/using-name-or-likeness-another

Using the Name or Likeness of Another
In most states, you can be sued for using someone else's name, likeness, or other personal attributes without permission for an exploitative purpose.

**Comment by the-machine-whisperer** [comment]


**Reply by allisoncfigus** [reply]
What caused you to come up with the word "Figus"? None of you seem to be named that. Even if you are, it is sort of harassment to the Figus family to imply that we endorse Figma. Are any of you named Figus and just using screennames? If so, then it goes into different territory.
